packages/astro/src/vite-plugin-markdown/index.ts

```diff
@@ -86,6 +84,8 @@ export default function markdown({ config, logging }: AstroPluginOptions): Plugi
 				};
 				export async function Content() {
 					const { layout, ...content } = frontmatter;
+					content.file = file;
```

Comment:
Fixes existing bug where `frontmatter.file` and `frontmatter.url` are added to `glob` results as well. This is inconsistent with the previous API and type defs.